kanidmd/src/lib/lib.rs

```diff
@@ -3,7 +3,7 @@
 #![deny(clippy::unwrap_used)]
 #![deny(clippy::expect_used)]
 // #![deny(clippy::panic)]
-#![deny(clippy::unreachable)]
+#![allow(clippy::unreachable)]
```

Comment:
We should deny unreachhable an annotate individual locations where we need to allow it I think.